mmcv/runner/ipu/hooks.py

```diff
@@ -0,0 +1,104 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from mmcv.utils import TORCH_VERSION, digit_version
+from mmcv.runner.hooks import HOOKS, OptimizerHook, LrUpdaterHook
+
+
+def wrap_lr_update_hook(lr_hook_class):
```

Comment:
Use `functools.wraps` in decorators.

Reply:
currently it is only used to wrap a function, no need to use as a decorator